_search-plugins/neural-search.md

```diff
@@ -196,7 +224,60 @@ GET my_index/_search
   }
 }
 ```
+{% include copy-curl.html %}
+
+### Setting a default model on an index or field
+
+To eliminate passing the model ID with each neural query request, you can set a default model on a k-NN index or a field. 
+
+First, create a [search pipeline]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/index/) with a [`neural_query_enricher`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/neural-query-enricher/) request processor. To set a default model on an index, provide the model ID in the `default_model_id` parameter. To set a default model on a specific field, provide the field name and the corresponding model ID in the `neural_field_default_id` map:
```

Comment:
In this sentence, can you please mention that if both default_model_id and neural_field_default_id is provided then priority would be given to neural_field_default_id .